test/unit/minimal-live-connect.spec.ts

```diff
@@ -3,10 +3,10 @@
 import jsdom from 'global-jsdom'
 import sinon from 'sinon'
 import { expect, use } from 'chai'
-import { MinimalLiveConnect, LiveConnect } from '../../src'
+import { MinimalLiveConnect, LiveConnect } from '../../src/index.js'
```

Comment:
Node resolution can't find `index` automatically?

Reply:
that should work, will put them back.

Reply:
nope https://stackoverflow.com/a/68621282